### ISSUE TITLE
Your Partial Conv is  new in computer vision. However, if you use a ground truth image in your loss function for your model trining, your paper is worthless for image inpainting. In most cases, we only have a deteriorated image, and the the ground truth is an unknown target. 

### ISSUE THREAD

**Issue by Flying-ow** [naoto0804/pytorch-inpainting-with-partial-conv#64]

**Comment by naoto0804** [comment]
Although it is not my model, you are right. If one doesn't know the exact regions to be inpainted, the task is called [blind image inpainting](https://arxiv.org/abs/2003.06816). It is much harder than the standard inpainting task, and the easiest way to solve that is by combining a region detector (or manual region identification) and an off-the-shelf standard inpainting model.

**Comment by Flying-ow** [comment]
After chatting with ChatGPT, I take back what I wrote about the paper for inpainting with a ground truth image in loss functions. Just like YOLO models, a trained inpainting model from the paper can be used for other deteriorated images, although we need a dataset with millions images and many days for model training. The algorithm presented by the paper is different from Deep Image Prior, in which a trained model can only be used for one image, but no training dataset is needed.

**Comment by Flying-ow** [comment]
After chatting with ChatGPT, I took back what I wrote about the paper for inpainting with a ground truth image in loss functions. Just like YOLO models, a trained inpainting model from the paper can be used for other deteriorated images, although we need a dataset with millions of images and many days for model training. The algorithm presented by the paper is different from Deep Image Prior, in which a trained model can only be used for one image, but no training dataset is needed.
________________________________
